src/ui/DiffWindow/index.js

```diff
@@ -36,15 +38,24 @@ async function initCodeMirror() {
     await Promise.all(promises);
     
     myCodeMirror = CodeMirror.MergeView(document.getElementById('conflictRes-Editor'), {
-        origLeft: origNote,
+        origLeft: remoteNote,
         mode: 'markdown',
         lineNumbers: true,
         connect: 'align',
         value: curNote,
         lineWrapping: true
     });
+
+    document.getElementById("titleLeft").value = remoteTitle;
+    document.getElementById("titleRight").value = curTitle;
 }
 
-// TODO: find a better way to wait for document load.
-// simple window.onload didn't work.
-setTimeout(initCodeMirror, 100);
\ No newline at end of file
+
+// see if DOM is already available
+if (document.readyState === "complete" || document.readyState === "interactive") {
+    // call on next available tick
+    // 1ms still didn't do the job. I had to make it 50ms
+    setTimeout(initCodeMirror, 50);
```

Comment:
Instead of setting an arbitrary timeout, why not poll for the whatever object you need? You could have an interval every x ms, that check for example for `typeof CodeMirror` (or whatever object you need). Then once it's no longer undefined, you clear the interval.

Reply:
You can see an example where I've done this. It's a bit over engineered, but works well.

https://github.com/CalebJohn/joplin-rich-markdown/blob/main/src/richMarkdown.ts#L130